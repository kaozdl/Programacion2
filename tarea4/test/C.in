encolar
desencolar 1
insertar_binario (15,rr) (12,z) (11,a) (10,b) .
desencolar 15
desencolar 1
Fin
