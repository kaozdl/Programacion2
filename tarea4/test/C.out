1>2>3>
-(12,z)
(15,rr)
--(10,b)
-(11,a)
2
1
4>5>6>Fin.
